core/src/test/java/com/bakdata/kafka/ProcessingErrorTest.java

```diff
@@ -85,10 +57,4 @@ void shouldAllowNullValue() {
                 .isNotNull();
     }
 
-    @Test
-    void shouldNotAllowNullDescription() {
```

Comment:
This test is missing in the avro module I think. Although it can also be part of this module as the description of DeadLetterDescription should be non nullable

Reply:
Introduced some Nullability checks in DeadLetterDescriptionTest...